From 7d75d3c12e7537b545cc5390db07eb2512d81494 Mon Sep 17 00:00:00 2001
From: Awais Belal <awais_belal@mentor.com>
Date: Thu, 1 Jun 2017 13:03:09 +0500
Subject: [PATCH 1/5] json: fix layer paths

This would allow picking up the layers when
they are installed to system's libdir.

Signed-off-by: Awais Belal <awais_belal@mentor.com>
---
 layersvt/linux/VkLayer_api_dump.json                   | 2 +-
 layersvt/linux/VkLayer_monitor.json                    | 2 +-
 layersvt/linux/VkLayer_screenshot.json                 | 2 +-
 vktrace/vktrace_layer/linux/VkLayer_vktrace_layer.json | 2 +-
 4 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/layersvt/linux/VkLayer_api_dump.json b/layersvt/linux/VkLayer_api_dump.json
index 4007dc013..10436be80 100644
--- a/layersvt/linux/VkLayer_api_dump.json
+++ b/layersvt/linux/VkLayer_api_dump.json
@@ -3,7 +3,7 @@
     "layer" : {
         "name": "VK_LAYER_LUNARG_api_dump",
         "type": "GLOBAL",
-        "library_path": "./libVkLayer_api_dump.so",
+        "library_path": "libVkLayer_api_dump.so",
         "api_version": "1.0.46",
         "implementation_version": "2",
         "description": "LunarG debug layer"
diff --git a/layersvt/linux/VkLayer_monitor.json b/layersvt/linux/VkLayer_monitor.json
index 662942bb2..c43fbde92 100644
--- a/layersvt/linux/VkLayer_monitor.json
+++ b/layersvt/linux/VkLayer_monitor.json
@@ -3,7 +3,7 @@
     "layer" : {
         "name": "VK_LAYER_LUNARG_monitor",
         "type": "GLOBAL",
-        "library_path": "./libVkLayer_monitor.so",
+        "library_path": "libVkLayer_monitor.so",
         "api_version": "1.0.46",
         "implementation_version": "1",
         "description": "Execution Monitoring Layer"
diff --git a/layersvt/linux/VkLayer_screenshot.json b/layersvt/linux/VkLayer_screenshot.json
index dddc4cb0f..f9fdb815f 100644
--- a/layersvt/linux/VkLayer_screenshot.json
+++ b/layersvt/linux/VkLayer_screenshot.json
@@ -3,7 +3,7 @@
     "layer" : {
         "name": "VK_LAYER_LUNARG_screenshot",
         "type": "GLOBAL",
-        "library_path": "./libVkLayer_screenshot.so",
+        "library_path": "libVkLayer_screenshot.so",
         "api_version": "1.0.46",
         "implementation_version": "1",
         "description": "LunarG image capture layer"
diff --git a/vktrace/vktrace_layer/linux/VkLayer_vktrace_layer.json b/vktrace/vktrace_layer/linux/VkLayer_vktrace_layer.json
index da4a5b1c4..6cf6144c9 100644
--- a/vktrace/vktrace_layer/linux/VkLayer_vktrace_layer.json
+++ b/vktrace/vktrace_layer/linux/VkLayer_vktrace_layer.json
@@ -3,7 +3,7 @@
     "layer" : {
         "name": "VK_LAYER_LUNARG_vktrace",
         "type": "GLOBAL",
-        "library_path": "../vktrace/libVkLayer_vktrace_layer.so",
+        "library_path": "libVkLayer_vktrace_layer.so",
         "api_version": "1.0.46",
         "implementation_version": "1",
         "description": "Vktrace tracing library",
-- 
2.11.1

